debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -29,35 +54,8 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import org.apache.kafka.connect.data.Field;
-import org.apache.kafka.connect.data.Schema;
-import org.apache.kafka.connect.data.SchemaBuilder;
-import org.apache.kafka.connect.errors.ConnectException;
-import org.apache.kafka.connect.source.SourceRecord;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import com.github.shyiko.mysql.binlog.event.deserialization.AbstractRowsEventDataDeserializer;
-import com.github.shyiko.mysql.binlog.event.deserialization.json.JsonBinary;
-
-import io.debezium.DebeziumException;
-import io.debezium.annotation.Immutable;
-import io.debezium.annotation.VisibleForTesting;
-import io.debezium.config.CommonConnectorConfig.BinaryHandlingMode;
-import io.debezium.config.CommonConnectorConfig.EventConvertingFailureHandlingMode;
-import io.debezium.config.Configuration;
-import io.debezium.connector.mysql.antlr.MySqlAntlrDdlParser;
-import io.debezium.connector.mysql.strategy.ConnectorAdapter;
-import io.debezium.data.Json;
-import io.debezium.data.SpecialValueDecimal;
-import io.debezium.jdbc.JdbcValueConverters;
-import io.debezium.jdbc.TemporalPrecisionMode;
-import io.debezium.relational.Column;
-import io.debezium.relational.Table;
-import io.debezium.relational.ValueConverter;
-import io.debezium.time.Year;
-import io.debezium.util.Loggings;
-import io.debezium.util.Strings;
+import static io.debezium.config.CommonConnectorConfig.EventConvertingFailureHandlingMode.FAIL;
+import static io.debezium.config.CommonConnectorConfig.EventConvertingFailureHandlingMode.WARN;
```

Comment:
Please do not reorder the imports, use `./mvnw clean install -DskipITs -DskipTests` to apply the code style to move these back to the proper order and squash the change.  Thanks.

Reply:
Thanks for your feedbacks! Done.